app/controllers/problems_searcher.rb

```diff
@@ -25,6 +25,10 @@ module ProblemsSearcher
       Array(Problem.find(err_ids))
     end
 
+    expose(:selected_problems_ids) do
+      selected_problems.map(&:id).map(&:to_s)
```

Comment:
ah that makes sense. this way job info can be serialized for out-of-process background workers like sideqik, etc.